doc/subsystems/networking/tc.rst

```diff
@@ -0,0 +1,27 @@
+.. _tc-support:
+
+Network Traffic Classification
+##############################
+
+Overview
+********
+
+`Traffic classification <https://en.wikipedia.org/wiki/Traffic_classification>`_
+is an automated process which categorises computer network traffic according to
```

Comment:
change ``which categorises`` to ``that categorizes``   (American English spelling)